content/04-postgresql/13-reading-and-querying-data/04-optimizing-postgresql.mdx

```diff
@@ -0,0 +1,663 @@
+state_change     |
+wait_event_type  | Activity
+wait_event       | LogicalLauncherMain
+state            |
+backend_xid      |
+backend_xmin     |
+query_id         |
+query            |
+backend_type     | logical replication launcher
+-[ RECORD 3 ]----+-------------------------------
+datid            | 13921
+datname          | postgres
+pid              | 836027
+leader_pid       |
+usesysid         | 10
+usename          | postgres
+application_name | psql
+client_addr      |
+client_hostname  |
+client_port      | -1
+backend_start    | 2022-11-06 20:20:18.273218+01
+xact_start       | 2022-11-06 20:39:01.207078+01
+query_start      | 2022-11-06 20:39:01.207078+01
+state_change     | 2022-11-06 20:39:01.207088+01
+wait_event_type  |
+wait_event       |
+state            | active
+backend_xid      |
+backend_xmin     | 762
+query_id         |
+query            | select * from pg_stat_activity
+backend_type     | client backend
+-[ RECORD 4 ]----+-------------------------------
+datid            |
+datname          |
+pid              | 1961
+leader_pid       |
+usesysid         |
+usename          |
+application_name |
+client_addr      |
+client_hostname  |
+client_port      |
+backend_start    | 2022-11-01 11:03:44.082354+01
+xact_start       |
+query_start      |
+state_change     |
+wait_event_type  | Activity
+wait_event       | BgWriterHibernate
+state            |
+backend_xid      |
+backend_xmin     |
+query_id         |
+query            |
+backend_type     | background writer
+-[ RECORD 5 ]----+-------------------------------
+datid            |
+datname          |
+pid              | 1960
+leader_pid       |
+usesysid         |
+usename          |
+application_name |
+client_addr      |
+client_hostname  |
+client_port      |
+backend_start    | 2022-11-01 11:03:44.082065+01
+xact_start       |
+query_start      |
+state_change     |
+wait_event_type  | Activity
+wait_event       | CheckpointerMain
+state            |
+backend_xid      |
+backend_xmin     |
+query_id         |
+query            |
+backend_type     | checkpointer
+-[ RECORD 6 ]----+-------------------------------
+datid            |
+datname          |
+pid              | 1962
+leader_pid       |
+usesysid         |
+usename          |
+application_name |
+client_addr      |
+client_hostname  |
+client_port      |
+backend_start    | 2022-11-01 11:03:44.082653+01
+xact_start       |
+query_start      |
+state_change     |
+wait_event_type  | Activity
+wait_event       | WalWriterMain
+state            |
+backend_xid      |
+backend_xmin     |
+query_id         |
+query            |
+backend_type     | walwriter
+```
+
+</details>
+
+**Note:** Using the `\gx` line termination sequence instead of the traditional semicolon (`;`) tells PostgreSQL to display to use expanded output mode for the current query.  This displays the columns and associated values for each record vertically instead of horizontally, which can improve readability in some cases.
```

Comment:
```suggestion
**Note:** Using the `\gx` line termination sequence instead of the traditional semicolon (`;`) tells PostgreSQL to display expanded output mode for the current query.  This displays the columns and associated values for each record vertically instead of horizontally, which can improve readability in some cases.
```